test/test_autograd.py

```diff
@@ -4129,6 +4130,863 @@ def fn(*inputs):
         setattr(TestAutogradDeviceType, test_name, do_test)
 
 
+class TestAutogradFunctional(TestCase):
+    def _assert_same_struct(self, res, base):
+        # base and res should be Tensors or tuple of Tensors containing the same thing
```

Comment:
Not the same thing, but rather, have the same sizes?

Reply:
Yes, that's a typo. Will change.